### PR TITLE
Read trusted path to certificates

### PR DESCRIPTION
**Pull request by raphael-theriault-swi**
This reads the contents of the file set as trusted path to the `certificates` option. It also defaults to using the appoptics certificate when an appoptics collector is used and no explicit trusted path was provided. Marked as draft as it is blocked by a release of the bindings.

### CODE REVIEW

File: lib/get-unified-config.js [hunk 1]
```diff
@@ -221,13 +222,30 @@ function getUnifiedConfig () {
...
+  // if no explicit trusted path has been provided also use the appoptics collector certificate
   //
   if (!('metricFormat' in results.global)) {
     if ('endpoint' in results.global && results.global.endpoint.includes('appoptics.com')) {
       // if a custom endpoint is provided and is an appoptics collector, only send TransactionResponseTime
       results.global.metricFormat = 1
+
+      if (!('certificates' in results.global)) {
+        results.global.certificates = require('./appoptics.crt')
+      }
```
Comment (cheempz) [inline lib/get-unified-config.js]: Minor, I know we shouldn't expect `metricsFormat` to be set since it's not advertised... but for the edge case (perhaps testing) where it is set, and we're pointing to the AO collector, seems this defaulting of certificate would be missed?  At the cost of duplicating that endpoint check I think it would be clearer if the certificate and metrics handling are separated.

**Review by cheempz — COMMENTED** [review]
Looks great pending my edge-case paranoia @raphael-theriault-swi!  Also, seeing the tests failed (but maybe only for DNS?).  Were you able to see if changing the domain for those tests made them work?  If not, I would even prefer temporarily disabling them instead of false alarm.

File: lib/get-unified-config.js (hunk 1 above)
Comment (raphael-theriault-swi) [reply lib/get-unified-config.js]: Good point, I separated the logic entirely

**Comment by raphael-theriault-swi** [comment]
Changing the domain we run DNS tests against to `example.com` seems to have fixed the issue. Probably a mix of how GitHub Actions sets up networking and the lack of a `AAAA` record for `solarwinds.com` not playing nicely.

**Review by cheempz — APPROVED** [review]
🎉 thanks for the revisit and test fix @raphael-theriault-swi!
